.travis.yml

```diff
@@ -8,13 +8,12 @@ install:
 env:
   global:
     - CC_TEST_REPORTER_ID=972a29fc370fae49a37220d1c23dafd0e71184217547376c7c495cf9670aa607
-    - CT_REGISTER_MODE=False
     - PORT=5037
     - API_VERSION=v1
     - LOCAL_URL=http://127.0.0.1:5037
-    - CT_URL=http://127.0.0.1:9000
-    - CT_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Im1pY3Jvc2VydmljZSIsImNyZWF0ZWRBdCI6IjIwMTYtMDktMTQifQ.IRCIRm1nfIQTfda_Wb6Pg-341zhV8soAgzw7dd5HxxQ
-
+    - GATEWAY_URL=http://127.0.0.1:9000
+    - MICROSERVICE_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Im1pY3Jvc2VydmljZSIsImNyZWF0ZWRBdCI6IjIwMTYtMDktMTQifQ.IRCIRm1nfIQTfda_Wb6Pg-341zhV8soAgzw7dd5HxxQ
+    - AWS_CLOUD_WATCH_LOGGING_ENABLED=False
```

Comment:
AWS logging should be enabled and mocked during test execution. The reference PR I provided covers this, so hopefully it shouldn't be too hard to implement here as well. Also, please make the equivalent change on the test docker compose file.

Reply:
Also: this .travis.yml still uses python 3.6. please update it accordingly.